mod/file/thumbnail.php

```diff
@@ -6,7 +6,12 @@
  */
 
 // Get engine
-require_once __DIR__ . '/../../vendor/autoload.php';
+$initialRoot = dirname(dirname(__DIR__));
+$backupRoot = dirname(dirname(dirname($initialRoot)));
+
+if (!include_once "$initialRoot/vendor/autoload.php") {
```

Comment:
This is a single file with quite small amount of logic. I don't think the vars matter here.